share/jupyter/voila/templates/lab/index.html.j2

```diff
@@ -46,7 +46,7 @@ var voila_process = function(cell_index, cell_count) {
     Therefore it is important to have the cell loop in the template.
     The issue for Jinja is: https://github.com/pallets/jinja/issues/1044
     #}
-    {%- for cell in cell_generator(nb, kernel_id) -%}
+    {%- for cell in cell_generator(nb) -%}
```

Comment:
So we don't need the kernel id in the jupyter-config-data?

I recall there was a comment by @maartenbreddels about this, but don't remember what.

Reply:
`jupyter-config-data` gets the `kernel_id` from `kernel_start()`.

Reply:
Deja vu feeling here :), I think it was a different PR. We do need the kernel_id, otherwise, we break the API, and we lose flexibility.

Reply:
:+1: